core/src/test/java/io/questdb/test/jit/CompiledFilterIRSerializerTest.java

```diff
@@ -847,9 +847,10 @@ public void testInSubSelect() throws Exception {
         serialize("asymbol in (select asymbol from tab limit 1)");
     }
 
-    @Test(expected = SqlException.class)
+    @Test
     public void testTimestampInLiteral() throws Exception {
         serialize("atimestamp in '2020-01-01'");
+        assertIR("(i64 1577836800000000L)(i64 atimestamp)(>=)(i64 1577923199999999L)(i64 atimestamp)(<=)(&&)(ret)");
```

Comment:
I'll add more tests using more complex range expressions

Reply:
Yes, more tests are certainly required, including tests that verify that we get the same query results with and without enabled JIT.

Reply:
Also, advanced intervals should be tested, e.g. `ts in '2023-12-20;15s'` or `ts in '2023-09-20T23:59:58;4s;-1d;7'`

Reply:
Done. Added IR tests for range expressions as well as a results test with and without JIT in `ParallelFilterTest`